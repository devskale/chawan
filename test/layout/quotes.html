<!DOCTYPE html>
<style>
div { quotes: "1" "" "2" "" "3" "" "4" "" "5" "" "6" "" "7" "" }
#inc::before { content: open-quote }
#dec::before { content: close-quote }
</style>
<q style="quotes: none">test</q>
<q style="quotes: '\\' '/'">test</q>
<q style="quotes: '\\' '/' 'a' 'b'"><q>test</q></q>
<div>
<span>
<div id=inc>inc</div>
<div id=inc>inc</div>
<div id=inc>inc</div>
<div id=inc>inc</div>
</span>
<div id=inc>inc</div>
<div id=inc>inc</div>
</div>
<div id=inc>inc</div>
