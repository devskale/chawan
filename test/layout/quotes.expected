test \test/ \atestb/
1inc
2inc
3inc
4inc
5inc
6inc
7inc
